giacomodeliberali/BeepBeep-StravaWorker
# Updated fetch runs routine according to the new api interface

how to manually test it:
check in /tmp/ if you have an old beepbeep.dataservice.db file and delete it

Add to the file key.sh
export DATA_SERVICE=http://localhost:5002
export STRAVA_TOKEN=<your token copied from the previous app>

execute key.sh (both in the dataservice and datapump terminals)
while running the dataservice
launch the fetch routine
and check at http://localhost:5002/users/1/runs
